### PR TITLE
Created new imageType input for error images

### PR DESCRIPTION
**Pull request by blackbaud-conorwright**
Created new image type input
Moved image loading to the sky-error-image component

Resolves: blackbaud/skyux2#2034
Docs: blackbaud/skyux2#2179

### CODE REVIEW

**Review by Blackbaud-SteveBrush — CHANGES_REQUESTED** [review]
We'll need a visual test for the custom error type so that it's easy to test (manually), and to have a screenshot of its current state.

**Comment by Blackbaud-SteveBrush** [comment]
We'll need a companion pull request into skyux2 to update the demo.

**Comment by blackbaud-conorwright** [comment]
Added the visual testing and created an issue for making demo changes after this merges: blackbaud/skyux2#2190

File: src/app/visual/error/error-visual.component.html [hunk 1]
```diff
@@ -57,6 +57,21 @@
...
+    <sky-error-image
+      imageType="broken"
+    >
+    </sky-error-image>
+    <sky-error-title>
+      custom title
+    </sky-error-title>
+    <sky-error-description>
+      custom description
+    </sky-error-description>
```
Comment (Blackbaud-SteveBrush) [inline src/app/visual/error/error-visual.component.html:74]: Let's throw the `<sky-error-action>` in here (like the other examples), just to be sure.

**Review by Blackbaud-SteveBrush — COMMENTED** [review]
Just one comment.

**Comment by Blackbaud-ToddRoberts** [comment]
How does this interact with errorType? If you set errorType and imageType, which one wins?

**Comment by blackbaud-conorwright** [comment]
@Blackbaud-ToddRoberts They aren't tied together and it will behave the same as it does right now when you provide a custom image and an errorType. It will show both images

**Review by Blackbaud-ToddRoberts — COMMENTED** [review]
Seems fine to me, I assume if no consumer has brought up the possibility of ending up with 2 images yet that it's not an issue.

**Comment by blackbaud-conorwright** [comment]
closed in favor of: https://github.com/blackbaud/skyux-errors/pull/7

**Pull request closed without merging**